### ISSUE TITLE
Cannot reproduce the result of resnet-based model on task of GTA2CityScapes 

### ISSUE THREAD

**Issue by LHXXHB** [wasidennis/AdaptSegNet#56]
Hi, I just run your codes and made no changes. I mean that I did not fix the 'bug' of ASPP layer mentioned in another issue and could get 42.35 of mIoU with the pre-trained multi-level model you provide [here](http://vllab.ucmerced.edu/ytsai/CVPR18/GTA2Cityscapes_multi-ed35151c.pth). But when I train from scratch, I could not get equally competitive results. For the multi-level setting, when using **only Imagenet pre-trained** resnet you provide in this [link](https://www.dropbox.com/s/87m2lnu3dxfy7o1/resnet101-caffe.pth?dl=0), the higheset mIoU is **40.61** at **95k** iteration, and when using **Imagenet+COCO pretrained** model you provide [here](http://vllab.ucmerced.edu/ytsai/CVPR18/DeepLab_resnet_pretrained_init-f81d91e8.pth), the best result is **41.99** at **60k** iteration.  As to the single-level setting, the best result of **only Imagenet pre-trained** model  is **41.4** at **40k** iteration. Although it is the exact value of mIoU reported in your paper, it is so weird to appear at 40k iteraton. I have not tried the single-level setting based on Imagenet+COCO pretrained model. So, I want to know do you use Imagenet+COCO pretrained resnet as the initialization for both multi-level and single-level setting and when the best result appears for the single-level setting? The results of my trials are showed in the following figure. Could you give me some suggestions to reproduce your result? Thanks!

![adaptseg](https://user-images.githubusercontent.com/23610775/58164825-1edb6700-7cb9-11e9-8616-48d5aa029c3b.png)

**Comment by wasidennis** [comment]
Thanks for reporting your results! We note that adversarial training is usually not stable, and we also did not fix the random seed, so the results may vary a bit and the best model may appear at different iterations. We have tried both Imagenet and Imagenet + coco pre-training, and found that for either single or multi-level model, the results are similar. However, using Imagenet + coco seems to give better convergence, while using Imagenet-only would require longer training time.

In your case, I wonder have you resolved this issue, maybe by re-training the model? Thanks!
